mule-extension-model-loader/src/main/java/org/mule/tooling/api/AstGenerator.java

```diff
@@ -84,7 +98,7 @@ public ArtifactAst generateAST(List<String> configs, Path configsPath) throws Fi
   }
 
   public ArrayList<ValidationResultItem> validateAST(ArtifactAst artifactAst) throws ConfigurationException {
-    ValidationResult result = MuleAstUtils.validate(artifactAst);
+    ValidationResult result = MuleAstUtils.validatorBuilder().build().validate(artifactAst);
     ArrayList<ValidationResultItem> errors = new ArrayList<ValidationResultItem>();
     ArrayList<ValidationResultItem> warnings = new ArrayList<ValidationResultItem>();
     result.getItems().forEach(v ->{if(v.getValidation().getLevel().equals(Level.ERROR)){errors.add(v);}else {warnings.add(v);}});
```

Comment:
Maybe using  `result.getItems().stream()collect(groupingBy(v -> v.getValidation().getLevel()))` is more readable.

Reply:
This is not part of this PR. Nevertheless I tried that line and it didn't work for me ( but I have no experience with the groupingBy). Maybe we can review this in the future